components/form/TaskForm.tsx

```diff
@@ -1,19 +1,20 @@
 import { useUpdateTasks } from '@/lib/hooks/useTasks';
 import { Project, Task } from '@/lib/types/apiTypes';
 import { FormBase } from '@/lib/types/forms';
-import { FormControl, FormLabel, Input, FormErrorMessage, Button, Box } from '@chakra-ui/react';
+import { FormControl, FormLabel, Input, FormErrorMessage, Button, Box, Flex, Checkbox } from '@chakra-ui/react';
 import React, { useState } from 'react';
 import ErrorAlert from '../common/ErrorAlert';
 
 type TaskFields = Partial<Task> & { project: Project };
 
 const validateTaskFields = (fields: TaskFields, projectId: number): Task => {
-    const { name, project } = fields;
-    if (name) {
+    const { name, project, billable } = fields;
+    if (name && billable) {
```

Comment:
This always fails when `billable` is `false`. Should have `billable !== undefined`